### DISCUSSION TITLE
Uploading mov fails from iOS app and from webUI

### DISCUSSION THREAD

**Discussion by szinn** [immich-app/immich#14397] [Q&A]
### The bug

I have successfully uploaded some movies but I have two that fail from both the app and from the UI. Tried turning on verbose logging, but there is no info why they failed

### The OS that Immich Server is running on

k8s on Talos

### Version of Immich Server

v1.121.0

### Version of Immich Mobile App

v1.121.0 build 184

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
https://raw.githubusercontent.com/szinn/k8s-homelab/refs/heads/main/kubernetes/main/apps/media/immich/app/helm-release.yaml
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

1. Upload file from web UI fails
...

**Comment by mmomjian** [comment]
This is likely an issue with your VPN, reverse proxy, or cloudflare tunnels.

**Reply by szinn** [reply]
This appears to be the issue. Moved immich to be internal to my network only and the uploads worked. Going through cloudflare tunnels was removed from the setup

**Reply by gadgetusaf** [reply]
Cloudflare tunnels limit the size of chunks to 100MB. Do I need to set a specific setting on the server? I understand that if I move this to my internal network, it works as well. but this wont work for family members.

**Reply by mmomjian** [reply]
There’s no setting - you can’t upload files over 100MB over CF tunnel.

**Comment by bo0tzz** [comment]
You'll wanna put a proxy-body-size annotation on your ingress, eg https://github.com/immich-app/immich-charts/blob/main/charts/immich/values.yaml#L81

**Reply by szinn** [reply]
I added that and no success - Also, that is set as a default in nginx config.

**Reply by gadgetusaf** [reply]
Cloudflare tunnels limit the size of chunks to 100MB.
